### PR TITLE
Chore/cleanup monitoring

### PR DESCRIPTION
**Pull request by davidzwa**
Restructure SSE and cleanup in panel, list, view before tackling #329 

- Rename client files
- Tested pages by hand
- Changelog added

### CODE REVIEW

File: client_src/js/camera-view.runner.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { setViewType } from "./monitoring/monitoring-view.state";
+import { createClientSSEWorker } from "./lib/client-worker";
+import {
+  monitoringSSEEventHandler,
```
Comment (NotExpectedYet) [inline client_src/js/camera-view.runner.js:4]: SSEE

File: client_src/js/lib/modules/currentOperations.js [hunk 2]
```diff
@@ -43,7 +43,6 @@ export default function currentOperations(
     });
   }
 
-  currentOperationsCount = currentOperationsCount;
```
Comment (NotExpectedYet) [inline client_src/js/lib/modules/currentOperations.js:46]: It does me make me wonder when I see stuff like this bahaha

Comment (davidzwa) [reply client_src/js/lib/modules/currentOperations.js:46]: JUST ASSIGN IT DAMMIT!

**Review by NotExpectedYet — APPROVED** [review]
lgtm

**Pull request merged**